source/boards/work-with-cards.rst

```diff
@@ -69,10 +47,46 @@ Work with property types
 * **Checkboxes** are useful for agenda items that are regularly revisited in weekly sprint or grooming meetings.
 * **Multi-select** allows you to create badges to indicate things like status.
 * **Numbers** are useful to capture metrics such as task sizing or effort estimates.
-* **Person** provides a quick way to capture user assignments.
+* **Person** provides a quick way to capture user assignments. Note that this is not available in Personal Desktop.
 * **URL** can be used to provide a link to a pull request or relevant website.
 * **Created time/Created by/Last updated time/Last updated by** are good ways order tasks in order of recency.
 
+Card badges
+-----------
+
+Card badges are a quick way to view card details without opening up a card. To add them, select **Properties > Comments and Description**. Icons related to the card description, comments, and checkboxes will be displayed on cards with the respective content. Open the card to view the details.
+- The description icon indicates that a card has a text description.
+- The comment icon displays a number indicating how many comments have been added to a card. When a new comment is added, that number is updated.
+- The checkbox icon displays the number of items checked off relative to the total number of checkboxes within the card. When an item is checked off, the icon is automatically updated.
```

Comment:
@justinegeffen Not sure what I did wrong here, but lines 58-60 are not previewing as a bulleted list.